supplementary/alica_ros1/alica_tests/src/test/test_alica_engine_plan_parser.cpp

```diff
@@ -165,418 +165,446 @@ class AlicaEngineTest : public AlicaTestFixture
-                case 1402488959965:
-                    checkState(s, 1402488959965, "GetBall", "", {}, {1402488991762}, {1402488990761, 1402489064693}, 1402488903550);
-                    break;
-                case 1402489037735:
-                    checkState(s, 1402489037735, "TryToDefendGoal", "", {1402489564599}, {1402489064693}, {1402489071510});
-                    break;
-                default:
-                    EXPECT_TRUE(false);
-                    cerr << "TEST_F(AlicaEngineTest, planParser) found a state not part of Defend.pml!" << endl;
-                    break;
+                cout << "EntryPoints: " << endl;
+                EXPECT_EQ(1u, plan->getEntryPoints().size()) << "Number of EntryPoints didnt fit AttackPlan.pml EntryPoints size." << endl;
+                for (const alica::EntryPoint* ep : plan->getEntryPoints()) {
+                    cout << "\t" << ep->getName() << " ID: " << ep->getId() << endl;
+                    checkEntryPoint(ep, 1402488646221, "MISSING_NAME", "", false, 0, 2147483647, 1402488646220, 1225112227903, "DefaultTask");
                 }
-            }
-            cout << "Transitions: " << endl;
-            EXPECT_EQ(4u, plan->getTransitions().size()) << "Number of Transitions didnt fit AttackPlan.pml EntryPoints size." << endl;
-            for (const alica::Transition* t : plan->getTransitions()) {
-                cout << "\t" << t->getName() << " ID: " << t->getId() << endl;
-                switch (t->getId()) {
-                case 1402488990761:
-                    checkTransition(t, 1402488990761, "TackleToGetBall", "GetBallToTackle", 1678986049909129132, 1402488959965, 1402488903549,
-                            "DefaultCondition", "", "", "DefaultPlugin", true);
-                    break;
-                case 1402488991762:
-                    checkTransition(t, 1402488991762, "TackleToGetBall", "TackleToGetBall", 1678986049909129132, 1402488903549, 1402488959965,
-                            "DefaultCondition", "", "", "DefaultPlugin", true);
-                    break;
-                case 1402489064693:
-                    checkTransition(t, 1402489064693, "GetBallToTryToDefendGoal", "TESTESTETS", 1678986049909129132, 1402488959965, 1402489037735,
-                            "DefaultCondition", "", "", "DefaultPlugin", true);
-                    break;
-                case 1402489071510:
-                    checkTransition(t, 1402489071510, "TryToDefendGoalToGetGoal", "TryToDefendGoalToGetGoal", 1678986049909129132, 1402489037735, 1402488910751,
-                            "DefaultCondition", "", "", "DefaultPlugin", true);
-                    break;
-                default:
-                    EXPECT_TRUE(false);
-                    cerr << "TEST_F(AlicaEngineTest, planParser) found a transition not part of Defend.pml!" << endl;
-                    break;
+                cout << endl;
+                break;
+            case 1402488893641:
```

Comment:
can't review this changes properly

Reply:
I didn't make any changes to this code. The only thing I did is to move the plan parsing checks into a function in the base class & call it in both tests